scopes/component/component-filters/envs-filter.tsx

```diff
@@ -83,9 +102,10 @@ function envsFilter({
 
   currentFilter.state.envsState.forEach((state) => {
```

Comment:
consider changing to `map` instead of `forEach`. better create a new one and can memoized it